src/DynamoCoreWpf/ViewModels/Core/DynamoViewModelEventArgs.cs

```diff
@@ -183,6 +183,12 @@ public class OpenNodeAnnotationEventArgs : OpenDocumentationLinkEventArgs
         /// Nodes category.
         /// </summary>
         public string Category { get; }
+
+        /// <summary>
+        /// Name of the package this node belongs to
+        /// </summary>
+        public string PackageName => Category.Split('.').FirstOrDefault();
```

Comment:
> is this always true, even when customizations are applied, is the package name always first?

I havent seen anything to indicate otherwise, but that might not mean that its 100% true. I dont think there are any APIs to get the package a node belongs to, so this seemed like the closest i could get.

Reply:
I think there is some logic in the workspace references to figure this out that you might be able to leverage.

Reply:
@mjkkirschner couldnt find anything in the WorkspaceDependencies, i did find a `GetNodePackage` in the workspaceModel but its private so will have to make it internal, you see any issues with that?
https://github.com/DynamoDS/Dynamo/blob/master/src/DynamoCore/Graph/Workspaces/WorkspaceModel.cs#L1800

Reply:
hmm, thats what I was thinking of as well, and no problem with making it internal, but looking at the code, I'm not sure it will work very well - can you give it a shot?

It's only going to be able to figure out the given package if the packages are loaded in the package manager... and I don't think that will be the case when the CLI tool runs - package manager extension won't even be loaded, will it?

Reply:
The CLI tool doesn't actually rely on this as this only for creating the `OpenNodeAnnotationEventArgs` (when users click `help` in the nodes context menu) that gets sent to the DocumentationBrowser to tell it what documentation to open. So it should be fine, did test it and seems to work fine.